code/controllers/subsystem/vote.dm

```diff
@@ -190,6 +190,18 @@ SUBSYSTEM_DEF(vote)
 				if(!lower_admin && SSmapping.map_voted)
 					to_chat(usr, span_warning("The next map has already been selected."))
 					return FALSE
+				var/list/previous_maps = list()
+				if(SSdbcore.Connect())
+					var/datum/DBQuery/query_previous_maps = SSdbcore.NewQuery({"
```

Comment:
Duplicate code

Reply:
Could maybe be split off into a proc on the mapping subsystem?